dusty/systems/docker/files.py

```diff
@@ -34,3 +34,8 @@ def move_file_inside_container(app_or_service_name, source_path, dest_path):
 
     _create_dir_in_container(client, container, parent_dir(dest_path))
     _move_in_container(client, container, source_path, dest_path)
+
+def container_path_exists(app_or_service_name, path):
+    client = get_docker_client()
```

Comment:
We should see if we can memoize this if we haven't already. Probably a separate PR, though.